src/crowdsourced/modal/CancelNotice.tsx

```diff
@@ -0,0 +1,19 @@
+import { FunctionComponent } from "react";
+
+export const CancelBookingNotice: FunctionComponent<{ className: string }> = ({
```

Comment:
We can probably use the existing version of this component, don't need to double up. Maybe move the original one to another folder?